### PR TITLE
HUM-1501 - web-accessibility improve tab links UI

### PR DESCRIPTION
**Pull request by andystu**

### CODE REVIEW

File: src/components/AoTable.vue [hunk 1]
```diff
@@ -378,8 +378,10 @@ $fake-scrollbar-color: $color-gray-90;
 .ao-table.ao-table--clickable {
   tbody > tr {
     cursor: pointer;
+    td > a{
+      color: #474a4c;
```
Comment (vinle173) [inline src/components/AoTable.vue:382]:
```suggestion
      color: $color-gray-10;
```

**Comment by vinle173** [comment]
what about the display for other table with link inside?
ex
Screen Shot 2020-08-05 at 1 13 41 PM

**Comment by andystu** [comment]
> what about the display for other table with link inside?
> ex
> Screen Shot 2020-08-05 at 1 13 41 PM

I think this question need to be confirmed with @dorianwolf.

**Comment by andystu** [comment]
after confirmed with @dorianwolf, this change might break other representation in other place, need to find other way to solve this issue.

**Pull request closed without merging**